lab-isaac/src/component/category-item/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import ExpenseItem from '../expense-item';
+import ExpenseForm from '../expense-form';
+import CategoryForm from '../category-form';
+import {expenseCreate} from '../../action/expense-actions';
+import {categoryUpdate, categoryDelete} from '../../action/category-actions';
+
+class CategoryItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      expenseForm: false,
+      categoryForm: false,
+    };
+
+    this.toggleExpense = this.toggleExpense.bind(this);
+    this.toggleCategory = this.toggleCategory.bind(this);
+  }
+
+  toggleExpense() {
+    this.setState({expenseForm: !this.state.expenseForm});
+  }
+
+  toggleCategory() {
+    this.setState({categoryForm: !this.state.categoryForm});
+  }
+
+  render() {
+    return (
+      <div className="category-item">
+        <div className="content-container">
+          <button className="remove" onClick={() => this.props.categoryDelete(this.props.category)}>Delete</button>
```

Comment:
you need something like an h2 here saying 'add an expense' because all that I can see is are two random buttons that say delete and new, it's not clear those are for expenses.